README.md

```diff
@@ -1,54 +1,99 @@
 
-The application can be packaged using:
-```shell script
-./mvnw package
+```console
+kubectl apply -f deploy/service.yaml
 ```
-It produces the `quarkus-run.jar` file in the `target/quarkus-app/` directory.
-Be aware that it’s not an _über-jar_ as the dependencies are copied into the `target/quarkus-app/lib/` directory.
+This should install the `notification-service` together with a Keptn `distributor` into the `keptn` namespace, which you can verify using:
 
-If you want to build an _über-jar_, execute the following command:
-```shell script
-./mvnw package -Dquarkus.package.type=uber-jar
+```console
+kubectl -n keptn get deployment notification-service -o wide
+kubectl -n keptn get pods -l run=notification-service
 ```
 
-The application is now runnable using `java -jar target/quarkus-app/quarkus-run.jar`.
+### Uninstall
 
-## Creating a native executable
+To delete a deployed *notification-service*, use the file `deploy/*.yaml` files from this repository and delete the Kubernetes resources:
 
-You can create a native executable using: 
-```shell script
-./mvnw package -Pnative
+```console
+kubectl delete -f deploy/service.yaml
 ```
 
-Or, if you don't have GraalVM installed, you can run the native executable build in a container using: 
-```shell script
-./mvnw package -Pnative -Dquarkus.native.container-build=true
+## Usage
+
+The goal of the *notification-service* is to inform the user about a new event via a notification in a chat app. (currently Slack but extendable)
+
+It needs the following configuration:
+
+### Kubernetes secrets
+
+* for Slack Environment
+
+  The Kubernetes secret `slack-access` containing the values `SLACK_TOKEN` and `SLACK_CHANNEL` is needed.
+
+* for Keptn Environment
+
+  The Kubernetes secret `keptn-access` containing the value `KEPTN_BRIDGE_DOMAIN` is needed.
+
+These variables can be set in the `service.yaml` file, but their name must not be changed!
+```yaml
+- name: SLACK_TOKEN
+  valueFrom:
+    secretKeyRef:
+      key: SLACK_TOKEN
+      name: slack-access
+- name: SLACK_CHANNEL
+  valueFrom:
+    secretKeyRef:
+      key: SLACK_CHANNEL
+      name: slack-access
+- name: KEPTN_BRIDGE_DOMAIN
+  valueFrom:
+    secretKeyRef:
+      key: KEPTN_BRIDGE_DOMAIN
+      name: keptn-access
 ```
 
-You can then execute your native executable with: `./target/quarkus-service-1.0.0-SNAPSHOT-runner`
+<!-- is an installation like https://github.com/keptn-sandbox/monaco-service/blob/main/README.md#option-1-monaco-projects-folders needed? -->
+
+## Development
+
+This is an open source project, so I welcome any contributions to make it even better!
+
+### Build yourself
+
+* Build the binary: `./mvnw package -Pnative` (with GraalVM) `./mvnw package -Pnative -Dquarkus.native.container-build=true` (without GraalVM)
+* Build not a binary: `./mvnw package`
+* Run tests: ``
+
+<!-- what name should the docker image have / the value before the '/' -->
```

Comment:
instead of `quarkus` we should use a placeholder, like `<your_docker_user>/notification-service`